spec/gorouter_templates_spec.rb

```diff
@@ -226,7 +226,8 @@
           'port' => 3745
         },
         'for_backwards_compatibility_only' => {
-          'empty_pool_response_code_503' => true
+          'empty_pool_response_code_503' => true,
+          'empty_pool_timeout' => 10
```

Comment:
here we should also put a dimension in to make it clear we are talking about a time.Duration element.
e.g.
```
'empty_pool_timeout' => '10s'
```